zebra-network/proptest-regressions/peer_set/set/tests/prop.txt

```diff
@@ -0,0 +1,7 @@
+# Seeds for failure cases proptest has generated in the past. It is
```

Comment:
please remove this file. it is generated when running tests locally. It will be nice to add this type of file regex to `.gitignore` but this is unrelated.